contracts/GSN/IRelayRecipient.sol

```diff
@@ -24,7 +42,32 @@ contract IRelayRecipient {
         view
         returns (uint256, bytes memory);
 
+    /**
+     * @dev Called by {IRelayHub} on approved relay call requests, before the relayed call is executed. This allows to e.g.
+     * pre-charge the sender of the transaction.
+     *
+     * `context` is the second value returned in the tuple by {acceptRelayedCall}.
+     *
+     * Returns a value to be passed to {postRelayedCall}.
+     *
+     * {preRelayedCall} is called with 100k gas: if it runs out during exection or otherwise reverts, the relayed call
+     * will not be executed, but the recipient will still be charged for the transaction's cost.
+     */
     function preRelayedCall(bytes calldata context) external returns (bytes32);
 
-    function postRelayedCall(bytes calldata context, bool success, uint actualCharge, bytes32 preRetVal) external;
+    /**
+     * @dev Called by {IRelayHub} on approved relay call requests, after the relayed call is executed. This allows to e.g.
```

Comment:
Is the sentence meant to have e.g. at the start rather than part way through?

```suggestion
     * @dev Called by {IRelayHub} on approved relay call requests, after the relayed call is executed. e.g. This allows to
```